modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,148 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management platform. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. To avoid the duplication, you must specify unique client IDs for your accounts. 
```

Comment:
Again, I do recommend moving this to the limitations section.

Reply:
Done! Thanks!